package.json

```diff
@@ -35,9 +35,9 @@
   "scripts": {
     "sequelize": "sequelize",
     "scrape": "node -e \"require('./src/server/services/scraper').scrape()\"",
-    "scrape:users": "node -e \"require('./src/server/services/scraper/users').scrape()\"",
-    "scrape:repos": "node -e \"require('./src/server/services/scraper/repos').scrape()\"",
-    "generate": "node -e \"require('./src/server/services/generator').generate()\"",
+    "scrape:users": "./src/server/services/scripts/generate-data.js --only users",
+    "scrape:repos": "./src/server/services/scripts/generate-data.js --only repos",
```

Comment:
@jouderianjr the scraper scripts are for another stuff, we don't want to replace it with _data generation_. Please, restore `scrape:users` and `scrape:repos`.